pyramid_oereb/standard/models/theme.py

```diff
@@ -0,0 +1,366 @@
+            office_id (str): The foreign key to the office which is responsible to this public law
+                restriction.
+            responsible_office (Office):
+                The dedicated relation to the office instance from database.
+            legend_entry_id (str): The foreign key to the legend entry this public law restriction is
+                related to.
+            legend_entry (pyramid_oereb.standard.models.airports_building_lines.LegendEntry):
+                The dedicated relation to the legend entry instance from database.
+        """
+        __table_args__ = {'schema': schema_name}
+        __tablename__ = 'public_law_restriction'
+        id = Column(String, primary_key=True, autoincrement=False)
+        law_status = Column(String, nullable=False)
+        published_from = Column(Date, nullable=False)
+        published_until = Column(Date, nullable=True)
+        view_service_id = Column(
+            String,
+            ForeignKey(ViewService.id),
+            nullable=False
+        )
+        view_service = relationship(
+            ViewService,
+            backref='public_law_restrictions'
+        )
+        office_id = Column(
+            String,
+            ForeignKey(Office.id),
+            nullable=False
+        )
+        responsible_office = relationship(Office)
+        legend_entry_id = Column(
+            String,
+            ForeignKey(LegendEntry.id),
+            nullable=False
+        )
+        legend_entry = relationship('LegendEntry', backref='public_law_restrictions')
+
+    class Geometry(Base):
+        """
+        The dedicated model for all geometries in relation to their public law restriction.
+        Attributes:
+            id (str): The identifier. This is used in the database only and must not be set manually. If
+                you  don't like it - don't care about.
+            law_status (str): The status switch if the document is legally approved or not.
+            published_from (datetime.date): The date when the geometry should be available for
+                publishing on extracts. This directly affects the behaviour of extract
+                generation.
+            published_until (datetime.date): The date from when the geometry should not be available
+                anymore for publishing on extracts. This directly affects the behaviour of extract
+                generation.
+            geo_metadata (str): A link to the metadata which this geometry is based on which delivers
+                machine  readable response format (XML).
+            public_law_restriction_id (str): The foreign key to the public law restriction this geometry
+                is  related to.
+            public_law_restriction (PublicLawRestriction): The dedicated relation to the public law
+                restriction instance from database.
+            office_id (str): The foreign key to the office which is responsible to this public law
```

Comment:
Is this made intentionally or is it took over from a former model. I think there is no such relation in the "Rahmenmodell".

Reply:
You are right. But for the sake of blocking I filed a new issue and we will solve this in a new PR https://github.com/openoereb/pyramid_oereb/issues/1310